src/client/index.js

```diff
@@ -807,8 +807,10 @@ window.TESTER = {
       setTimeout(() => {
         navigator.getVRDisplays().then(displays => {
           var device = displays[0];
-          //if (device.isPresenting) device.exitPresent();
-          device.requestPresent( [ { source: canvas } ] );
+          if (device) {
```

Comment:
In my Chrome, somehow `getVRDisplays()` returns `[undefined]` in test. Perhaps WebVR emulation doesn't work correctly in "test" while working fine in "launch"?

```
navigator.getVRDisplays().then(displays => {
  console.log(displays);
});
-> [undefined]
```

Then checking `device` here instead of `displays.length > 0`.

Reply:
This issue seems to have been somehow solved? Here it returns `[VRDisplay]` now.